### ISSUE TITLE
Radio button for 'thumbnail' in file manager displays on wrong line

### ISSUE THREAD

**Issue by hackmastera** [projecthydra-labs/hyrax#928]
### Descriptive summary

Hyrax master; Chrome 58.0.3029.96

### Expected behavior

Radio button should be next to 'thumbnail'

### Actual behavior

Radio button is up by filename box.

### Steps to reproduce the behavior

1. Create a work with two files
1. Go to the file manager

![screen shot 2017-05-09 at 2 32 47 pm](https://cloud.githubusercontent.com/assets/845363/25869147/ec621b4a-34c4-11e7-9f52-f6f930b9eac2.png)

**Comment by andjsmit** [comment]
So it is not a simple matter of alignment after all. It is actually structured to have radio buttons in front of BOTH the Thumbnail and Representative Media. Changing the CSS of the radio buttons from absolute to relative positioning displays the button properly. The question is then should both fields be displayed. I'm going to experiment with what choosing different selections for each results in.

**Comment by hackmastera** [comment]
@andjsmit Yes both should be displayed. The thumbnail shows on the results page. The representative media shows on the show page. The use case that's been cited to me is, e.g. an audio file where you want to allow the audio file as a download but show the thumbnail in search results.

Fixing the alignment is sufficient to close this bug. Thanks!!!

**Comment by andjsmit** [comment]
@hackmastera The other issue I'm seeing is that changing the radio selections does not enable the save button like happens when the order of pages is changed.

**Comment by andjsmit** [comment]
I'll write up a new ticket for the save button issue once alignment fix is in place, unless there is one already.

**Comment by mjgiarlo** [comment]
@andjsmit I don't believe there is one already. Thanks!
